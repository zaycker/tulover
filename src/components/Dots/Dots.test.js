import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Dots from './index';

it('render correctly Dots component', () => {
    const DotsComponent = renderer.create(
        <Dots elements={[1,2,3]} selected={1} className="example" />
    ).toJSON();
    expect(DotsComponent).toMatchSnapshot();
});

it('should use props [elements, selected, className]', () => {
    const wrapper = shallow(
        <Dots elements={[1,2,3]} selected={1} className="example" />
    );
    expect(wrapper.find('.example')).toHaveLength(1);
    expect(wrapper.find('.dots__dot')).toHaveLength(3);
    expect(wrapper.find('.dots__dot').at(1).is('.dots__dot_active')).toBe(true);
});