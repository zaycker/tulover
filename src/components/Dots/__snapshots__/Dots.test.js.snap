// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Dots component 1`] = `
<div
  className="dots example"
>
  <span
    className="dots__dot"
  />
  <span
    className="dots__dot dots__dot_active"
  />
  <span
    className="dots__dot"
  />
</div>
`;
